libs/build_cmakemodules.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eux
+
+name="cmakemodules"
+repo=${1:-${STACK_cmakemodules_repo:-"NOAA-EMC"}}
+version=${2:-${STACK_cmakemodules_version:-"develop"}}
+id=${version//\//-}
+
+if $MODULES; then
+  set +x
+  source $MODULESHOME/init/bash
+  module list
+  set -x
+
+  prefix="${PREFIX:-"/opt/modules"}/core/$name/$repo-$id"
```

Comment:
should be just `$id`

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884